subgraph.rinkeby.yaml

```diff
@@ -0,0 +1,135 @@
+specVersion: 0.0.2
+schema:
+  file: ./schema.graphql
+dataSources:
```

Comment:
I see this got into the .gitignore file, but it also ended up in the repo! :D

Reply:
fixed!